packages/story-editor/src/utils/getBlurHashFromImage.js

```diff
@@ -37,7 +37,12 @@ const getImageData = (image) => {
 };
 
 const getBlurHashFromImage = async (src) => {
-  const image = await preloadImage(src).catch(() => {});
+  let image;
+  try {
+    image = await preloadImage(src);
+  } catch (error) {
+    //ignore
+  }
   const imageData = getImageData(image);
```

Comment:
Wouldn't this also throw an error when passing `null` as `image`?

Would be great if you could actually test out if these cases work as expected when you purposefully trigger an error in `preloadImage`.

Reply:
@miina this wont cause a problem because it is being used at only one place that it in `useDetectBlurHash` which already wraps `getPreloadImage` in a try catch thinking of it now I think should this even be wrapped in try catch since useDetectBlurHash wraps it as a whole?

Reply:
Since this is an util that could in theory be used elsewhere as well, then to be on the safe side, I think it's still better to use `try` here as well, not assuming the util will only ever be used in one place. Since the try-catch around using this util inside `useDetectBlurHash` is used for `getBlurHashFromImage` only (as `saveBlurHash` has its own error handling), might make more sense to remove it from there instead but still keep it in the util.

Reply:
Leaving getBlurhashFromImage with a try/catch

Reply:
Ok, but let's make sure `getImageData` doesn't trigger error here then as well if `preloadImage` fails.

Reply:
(Marking it back to unresolved since currently, `getImageData` would still error if we pass `undefined` image there.)

Reply:
It will now return a `rejected` Promise if getPreloadImage fails to load the image